boards/renesas/rsk_rx130_512kb/Kconfig.rsk_rx130_512kb

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_RSK_RX130_512KB
```

Comment:
why have you used 512kb in the name, is this the actual name of the board you sell "rsk rx130 512kb"?

Reply:
@nordicjm , yes on the web site its name is Renesas Starter Kit RX130 512kb.
https://www.renesas.com/en/products/microcontrollers-microprocessors/rx-32-bit-performance-efficiency-mcus/rx130-512kb-starter-kit-renesas-starter-kit-rx130-512kb?doc_secondary=visible

There is another board name RSK RX130 which only has 128kb of ROM
https://www.renesas.com/en/products/microcontrollers-microprocessors/rx-32-bit-performance-efficiency-mcus/rx130-128kb-starter-kit-renesas-starter-kit-rx130-128kb-non-promotion?srsltid=AfmBOorl8HwnTB3GbXUaUvPV0ZDL4lwUP2lzebUCAQKsLAsye_hO6Qo4

We also use this naming to be unify with the Renesas e2studio naming of the board
![image](https://github.com/user-attachments/assets/cb3829df-be91-49cd-bcd4-975c10c1eec9)

Reply:
There are board revisions in zephyr, so you could have `rsk_rx130@512kb` and `rsk_rx130@128kb` if you wanted to which means you can have common files e.g. devicetree then have overlayes which provide the different parts, or they can be kept as separate boards.